content/docs/12-enterprise-version/03-deploying-palette-with-helm.md

```diff
@@ -252,7 +264,7 @@ You now have a self-hosted instance of Palette installed in a Kubernetes cluster
   <br />
 
   ```shell
-  helm install palette /path/to/chart.tgz --file /path/to/values.yaml
+  helm install palette /path/to/chart.tgz -f /path/to/values.yaml
```

Comment:
@karl-cardenas-coding Don't we typically use the long form of the command options?